src/components/new-request/search.vue

```diff
@@ -365,6 +402,7 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
   activeActionGroup = NaN
   showRequestActionTooltip = false
   business = null as BusinessSearchIF
+  isPassValidation = false
```

Comment:
This is weird because its value is set in a computed (line 427 below), which may not be evaluated when you think. (Computed is evaluated when it's first used, and thereafter if anything it uses has changed.)

Can this be changed to a computed also?

Reply:
Did this get cleaned up?

Reply:
I see you changed this to a getter (computed). Looks good. Resolving.